### DISCUSSION TITLE
Shorten URL of images

### DISCUSSION THREAD

**Discussion by linax777** [Dan6erbond/share-me#25] [Ideas]
the direct link URL of images are too long to share with string, I would like short URL with no or configurable prefix .

**Comment by Dan6erbond** [comment]
This is an interesting idea. I feel like there would be two ways to go about it, either using the approach [GoS3](https://github.com/stv0g/gose) takes and integrating with existing URL shorteners. This would allow people to have uniform short links across their apps and domains, or integrate it into the API directly using links like `https://shareme.example.com/img/<file-id>`.

Both have pros and cons. The first is more modular, while the second follows a similar schema to Imgur, which is the app Share Me is inspired by. I'm looking for some more discussion towards this topic!
